aiida/backends/sqlalchemy/models/node.py

```diff
@@ -162,7 +162,7 @@ def get_aiida_class(self):
                                  "not valid: '{}'".format(self.pk, self.type))
 
         try:
-            PluginClass = load_plugin(Node, 'aiida.orm', pluginclassname)
+            PluginClass = load_plugin_safe(Node, 'aiida.orm', pluginclassname)
```

Comment:
You can remove it from the try/except, as the _safe version does not raise anymore